otx/recipes/stages/segmentation/selfsl.py

```diff
@@ -16,7 +16,17 @@
 
 optimizer = dict(type="SGD", lr=0.001, momentum=0.9, weight_decay=10e-4)
 
-lr_config = dict(policy="customstep", by_epoch=True, gamma=1, step=10)
```

Comment:
Is this okay to delete this config? @sungchul2 It looks like self supervised learning uses custom step hook for default lr config.

Reply:
Are there other step decay functions to replace `customstep`?
If not, experiments for performance comparison are required, and I think the change may be okay if there is no difference.

Reply:
I think it can be replaceable to mmcv StepLrUpdaterHook. Could you replace this config to use mmcv StepLrUpdaterHook? @cih9088

`lr_config = dict(policy="step", by_epoch=True, gamma=1, step=10)
`

Reply:
https://github.com/openvinotoolkit/training_extensions/pull/2111/commits/9a138d8f42a7d07e6f36c04491deb5c089ceb54d